mediaire_toolbox/queue/redis_wq.py

```diff
@@ -94,16 +97,88 @@ def _lease_exists(self, item):
     def put(self, item):
         self._db.lpush(self._main_q_key, item)
 
+    @staticmethod
+    def _get_timestamp(timeunit):
+        if timeunit == 'sec':
```

Comment:
let's comment a bit on this method as intuitively when one sees "get_timestamp" one thinks about a full timestamp e.g. in millisecond. Also, not sure about the name since we're simply returning a "part" of a timestamp, but also fine to keep it like that.

Reply:
since we are using the returned number eventually as a key, perhaps get_limit_key?